cmd/root.go

```diff
@@ -43,12 +44,14 @@ func Execute() {
 
 var rootCmdUser string
 var rootCmdPassword string
+var rootCmdPasswordFile string
 var rootCmdInsecure bool
 var rootCmdVerbose bool
 
 func init() {
 	RootCmd.PersistentFlags().StringVarP(&rootCmdUser, "user", "u", "", "use the specified username")
 	RootCmd.PersistentFlags().StringVarP(&rootCmdPassword, "password", "p", "", "use the specified password")
+	RootCmd.PersistentFlags().StringVarP(&rootCmdPasswordFile, "password-file", "", "", "use the password found in the specified file")
```

Comment:
The third argument to StringVarP should be a shorthand letter. Use "P" or call StringVar.